guides/doc-Planning_for_Project/topics/Provisioning_Concepts.adoc

```diff
@@ -33,7 +33,7 @@ To provision machines using PXE booting, ensure that you meet the following requ
 .Client requirements
 
 * Ensure that all the network-based firewalls are configured to allow clients on the subnet to access the {SmartProxy}.
-For more information, see xref:figu-Satellite_Topology_with_Isolated_Capsule[].
+For more information, see xref:{SmartProxy}-Networking_planning[].
```

Comment:
IMHO if we use "xref:" for links, then we can _always_ also use "_{context}" (if the anchor permits this).

Maybe a quick follow-up PR is the way to bring all anchors up to our usual conventions?

Reply:
This concerns the Deployment Planning part, that will be reviewed in another PR.